NebulaPatcher/Patches/Dynamic/BuildTool_Inserter_Patch.cs

```diff
@@ -0,0 +1,67 @@
+﻿using HarmonyLib;
+using NebulaModel.Packets.Factory;
+using NebulaWorld;
+using NebulaWorld.Factory;
+using System.Collections.Generic;
+
+namespace NebulaPatcher.Patches.Dynamic
+{
```

Comment:
There is a lot of redundant code between this, BuildTool_Path_Patch.cs, and BuildTool_Inserter_Patch.cs, I am working on merging this code.

Also these files are already in the branch so I'm not sure why they are being detected in the PR like this